src/en/developer-debugging.md

```diff
@@ -29,9 +29,26 @@ logs](./troubleshooting-logs.html#the-debug-log-command) section for more
 details and [filtering](./troubleshooting-logs.html#advanced-filtering) options
 with the `juju debug-log` command.
 
-You can also view Juju logs logs on an individual machine by using
-`juju ssh` to login to the unit in question and manipulate the log files in the
-`/var/log/juju` directory.
+### Log file location
+
+Since log messages stream in real time, it is possible to miss messages while
+using the debug-log command. In the case where you need to view the log files
+you can ssh to the machine and view the log files. To access the individual
+machine use `juju ssh <machine-number>` to get access to the machine and the
+Juju log files can be found in the `/var/log/juju` directory.
+
+### Log files on the controller
+
+In Juju 2.0+ the machine running the  the controller is not represented in the
+Juju model and not accessible by machine number. If you need the log files from
+the controller you must switch to the "admin" model and ssh to machine 0.
+
+```
+juju switch local.amazon:admin
+juju ssh 0
+```
```

Comment:
@AdamIsrael the `tail` command is why you are there in the first place (because you missed the log message)! I would resist telling someone how to view the logs because there are so many different ways and reasons to be viewing the logs (more|less|grep|vi|nano|emacs).